### PR TITLE
perlPackages.LinuxInotify2: add description/license to meta

### PR DESCRIPTION
**Pull request by zakame**
###### Motivation for this change

Applying my suggestion from https://github.com/NixOS/nixpkgs/pull/101122#discussion_r508457565 to update package meta/documentation.  No functional changes.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/top-level/perl-packages.nix [hunk 1]
```diff
@@ -11014,8 +11014,10 @@ let
     };
     propagatedBuildInputs = [ commonsense ];
 
-    meta = with stdenv.lib; {
-      platforms = platforms.linux;
+    meta = {
+      description = "Scalable directory/file change notification for Perl on Linux";
+      license = with stdenv.lib.licenses; [ artistic1 gpl1Plus ];
```
Comment (stigtsp) [inline pkgs/top-level/perl-packages.nix:11019]:
```suggestion
      license = with stdenv.lib.licenses; [ artistic1 gpl1Plus ];
```

I'm unable to find any reference of a license for this module. Even metacpan says "License: unknown".

Comment (zakame) [reply pkgs/top-level/perl-packages.nix:11019]: For MLEHMANN's modules he usually just says this under https://metacpan.org/source/MLEHMANN/Linux-Inotify2-2.2/COPYING

```
This module is licensed under the same terms as perl itself.
```

Similar with https://metacpan.org/release/AnyEvent ; I believe only Coro and EV have some exceptions.

Comment (stigtsp) [reply pkgs/top-level/perl-packages.nix:11019]: Thanks for clarifying

**Review by stigtsp — APPROVED** [review]
LGTM

**Comment by zakame** [comment]
Thanks! :+1:

**Pull request merged**